src/services/user/hooks/userService.js

```diff
@@ -379,6 +379,92 @@ const handleClassId = (hook) => {
 		.then((res) => Promise.resolve(hook));
 };
 
+const getFederalStatesRoles = async (app, roleNames) => {
+	let roles;
+
+	try {
+		roles = await app.service('roles').find({
+			query: {
+				name: { $in: roleNames },
+			},
+		});
+		return roles.data;
+	} catch (e) {
+		logger.error(e);
+		throw new GeneralError(`Could not find federalStates roles`);
+	}
+};
+
+const isStudent = async (app, roles) => {
+	if (roles.length === 0) {
+		return false;
+	}
+	try {
+		const studentRole = await app.service('roles').find({
+			query: {
+				_id: { $in: roles },
+				name: 'student',
+			},
+		});
+		return studentRole.data.length > 0;
+	} catch (e) {
+		logger.error(e);
+	}
+	return false;
+};
+
+const getSchoolState = async (app, schoolId, statesAbb) => {
```

Comment:
Maybe use `statesAbbr`, `statesAbbrv`, or `statesAbbrev` instead of `statesAbb`.

I will prefer `stateCodes`, but since the column is called `abbreviation`... :-)